### PR TITLE
EN6-138 Normalizing attributes names to match FE expectations

### PR DESCRIPTION
**Pull request by BrenoQVDS**
Normalized filters names to match FE expectations, now its possible to filter usings "id" and "description"

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Restest this please

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 1]
```diff
@@ -857,6 +858,50 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+                        .param("filters[0].attribute", "id")
+                        .param("filters[0].value", "EVN194")
+                        .sessionAttr("user", user)
+                        .header("Authorization", "Bearer " + accessToken));
+        String bodyResult = result.andReturn().getResponse().getContentAsString();
+        result.andExpect(status().isOk());
+        String expectedContentId = "EVN194";
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        String extractedId = JsonPath.read(bodyResult, "$.payload[" + 0 + "].id");
+        Assert.assertEquals(expectedContentId, extractedId);
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Check code style for building mvc test rules on my latest commit on PR #556

**Comment by entando-service** [comment]
All is Well

**Pull request merged**